src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -827,6 +827,56 @@ jit_floor_mod_emitter::jit_floor_mod_emitter(jit_generator_t* host,
     prepare_table();
 }
 
+///  Greater ///
+jit_greater_emitter::jit_greater_emitter(jit_generator_t* host, cpu_isa_t host_isa, const element::Type exec_prc)
```

Comment:
Also May I ask you to move you changes before `jit_gelu_erf_emitter` implementations in this file please? Because currently your code is within the class `jit_floor_mod_emitter` implementations :)